src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h

```diff
@@ -105,22 +100,16 @@ namespace OpenMS
     virtual void init(const Container& c)
     {
       computeSTN_(c);
-      is_result_valid_ = true;
     }
 
     ///Return to signal/noise estimate for date point @p index
-    ///@note the first query to this function will taake longer, as
+    ///@note the first query to this function will take longer, as
```

Comment:
this is not correct anymore, right? Since everything is computed during the call to `init()`.